### PR TITLE
Soften the pronouncements made in the README

### PR DESCRIPTION
**Pull request by pradyunsg**
This is more reflective of reality and has, uhm, less "flair" in a good way. :)

I've not added additional references or significantly rephrased any of the later parts of the README. If someone wants that, that can be a follow up.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,19 +1,18 @@
 Pipfile: the replacement for `requirements.txt`
 ===============================================
 
-``Pipfile`` and its sister ``Pipfile.lock`` are a replacement for the existing standard `pip`_'s ``requirements.txt`` file.
+``Pipfile`` and its sister ``Pipfile.lock`` are designed as a proposed replacement for an existing standard: `pip`_'s ``requirements.txt`` file.
```
Comment (uranusjr) [inline README.rst:4]: Maybe say an existing "format"? requirements.txt is not really a standard.

**Review by uranusjr — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (pradyunsg) [reply README.rst]: Done!

**Comment by pradyunsg** [comment]
Thanks for the review @uranusjr! ^.^

**Pull request merged**